plugins/couchbase/src/main/resources/plugin.properties

```diff
@@ -0,0 +1,3 @@
+service.default org.radargun.service.CouchbaseService
```

Comment:
For me, it is missing the `CouchbaseServerService` class implementation. For example, I would like to run RadarGun and compare the project X with Couchbase in a perf lab. Without `CouchbaseServerService` I will need to setup the server everytime. I saw https://docs.couchbase.com/server/4.1/getting-started/installing.html that we can download and install the server. A good entry point is https://github.com/radargun/radargun/blob/master/plugins/etcd/src/main/java/org/radargun/service/EtcdServerService.java